libs/blocks/graybox/graybox.css

```diff
@@ -190,7 +190,7 @@
 }
 
 .dialog-modal.graybox-modal {
-  z-index: var(--above-all);
+  z-index: calc(var(--above-all) + 1);
```

Comment:
Shouldn't the above all value be updated instead?

Reply:
That value is also used in the desktop view. If we only update the above-all, the bug will persist.

Reply:
That will cause a different bug in the desktop view, where the highlighted sections will cover the gnav when scrolling.